docs/guides/integrating-optimize.md

```diff
@@ -0,0 +1,122 @@
+---
+id: integrating-optimize
+title: Integrating Optimize
+sidebar_label: Integrating Optimize
+---
+
+## Purpose
+
+The following document provides a basic end-to-end glance into Optimize and its features for new business users.
+
+Optimize offers business intelligence tooling for Camunda enterprise customers. By leveraging data collected during process execution, you can access reports, share process intelligence, analyze bottlenecks, and examine areas in business processes for improvement.
+
+With Optimize, you can review heatmap displays for a closer look at the number of instances that took longer than average, based on duration distribution. As a result, reap actionable insights and rapidly identify the constraints of your system.
+
+See [What is Optimize](https://camunda.com/products/camunda-platform/optimize/reports/) for an in-depth overview of Optimize’s capabilities.
+
+[//]:# (The above is just a placeholder link. Amara, should we add a landing page for Optimize? This was the text I moved below this original Optimize guide.)
+
+## Set up 
+
+A Camunda Cloud user can launch Optimize from the Cloud Console — the interface where users create clusters, and launch both Operate and Tasklist. Therefore, ensure you’ve [created a Camunda Cloud account](./getting-started/create-camunda-cloud-account.md) before getting started with Optimize for SaaS users.
+
+So long as a user chooses a version of Zeebe when creating a cluster that includes Optimize, the user can access Optimize.  While creating a cluster, the creation modal will note if Optimize is included by each version of Zeebe that can be selected.
+
+[//]:# (Need to clarify above, what version(s) of Zeebe are applicable with Optimize, or is there an evergreen link we could add here?)
```

Comment:
I always vote for evergreen so something like "Camunda Cloud 1.2+" would be sufficient for me. If we need to be more specific in CCSM content, we can handle that separately. just confirm when Optimize was available for Camunda Cloud (SaaS).

Reply:
Resolved in latest commit.